PowerUp/app/src/main/res/layout/pgame.xml

```diff
@@ -54,14 +55,74 @@
         android:layout_alignParentRight="true"
         android:layout_below="@id/scenarioNameEditText"
         android:src="@drawable/marco" />
-
-    <ImageView
-        android:id="@+id/powerBarView"
-        android:layout_width="wrap_content"
+    <RelativeLayout
+        android:id="@+id/power_bar"
+        android:layout_width="match_parent"
         android:layout_height="@dimen/power_bar_view_height"
-        android:layout_alignParentBottom="true"
-        android:layout_alignParentLeft="true"
-        android:background="@drawable/clothing_room_all" />
+        android:layout_alignParentBottom="true">
+        <ImageView
+            android:id="@+id/powerBarView"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/power_bar_view_height"
+            android:layout_alignParentBottom="true"
+            android:layout_alignParentLeft="true"
+            android:background="@drawable/power_status_bar"
+            android:layout_alignParentRight="true"
+            android:layout_alignParentEnd="true" />
+        <com.akexorcist.roundcornerprogressbar.RoundCornerProgressBar
+            android:id="@+id/power_bar1"
```

Comment:
I think it would be better to give the bars the name of their power: `power_bar_healing` for example. Powers are defined in the [Score class](https://github.com/systers/powerup-android/blob/master/PowerUp/app/src/main/java/powerup/systers/com/datamodel/Score.java). Other than that, the rest looks good :)